src/batch/normalization/repositories/gateways/dbsderApi.spec.ts

```diff
@@ -0,0 +1,87 @@
+import {
+  BadRequestException,
+  HttpStatus,
+  ServiceUnavailableException,
+  UnauthorizedException
+} from '@nestjs/common'
+import { DbSderApiGateway } from './dbsderApi'
+import { MockUtils } from '../../../../shared/infrastructure/utils/mock.utils'
+import axios from 'axios'
+
+jest.mock('../../index', () => ({
+  logger: {
+    log: jest.fn(),
+    error: jest.fn()
+  },
+  normalizationContext: {
+    start: jest.fn(),
+    setCorrelationId: jest.fn()
+  }
+}))
+jest.mock('axios')
+const mockedAxios = axios as jest.Mocked<typeof axios>
+
+describe('DbSderApi', () => {
+  const mockUtils = new MockUtils()
+  const gateway = new DbSderApiGateway()
+  it('Returns 201 - if  dbSder API is called with valid parameters', async () => {
```

Comment:
jusqu'ici, on a plutôt rédigé sous la forme `it('returns 201 CREATED when etc...')`, du coup serais-tu ok pour remplacer les "if dbSder API" par "when dbSder API" ?

Reply:
Autre remarque : la description du test ne correspond pas à ce qui est fait dans le test :) 
Je m'explique : le test annonce une 201 si tous les paramètres sont ok (comme pour un test d'intégration), sauf qu'on est sur un TU et qu'on vérifie le retour du saveDecision, qui ne retourne pas de 201 :p